src/psyclone/tests/gocean1p0_test.py

```diff
@@ -276,12 +276,11 @@ def test_scalar_float_arg(tmpdir):
         "      DO j=1,jstop+1\n"
         "        DO i=1,istop+1\n"
         "          CALL bc_ssh_code(i, j, a_scalar, ssh_fld%data, "
-        "ssh_fld%grid%tmask)\n"
+        "ssh_fld%grid%subdomain%internal%xstop, ssh_fld%grid%tmask)\n"
```

Comment:
As previously commented, I don't understand why we're now passing `xstop` to this kernel?

Reply:
As discussed, I added this as an additional (originally failing) test case.